py-polars/tests/unit/streaming/test_streaming.py

```diff
@@ -331,6 +331,20 @@ def test_streaming_11219() -> None:
     ).collect(streaming=True).to_dict(as_series=False) == {"b": ["afoo", "cfoo", None]}
 
 
+def test_streaming_csv_headers_but_no_data_13770(tmp_path: Path) -> None:
```

Comment:
Can you mark this one with the `write_disk` pytest marker?

Reply:
done, thanks